ui/components/app/custom-spending-cap/index.scss

```diff
@@ -0,0 +1,27 @@
+.custom-spending-cap {
+  &__input-value-and-token-name {
+    display: contents;
+  }
+
+  &__input {
+    width: 100%;
+
+    &--button {
+      background: none;
+      color: var(--color-primary-default);
+    }
+
+    &--max-button {
+      color: var(--color-text-alternative);
+      background: none;
+      position: fixed;
```

Comment:
I think you want absolute positioning here. Also will need relative positioning for the wrapping container

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15522/commits/e029de4c594789c4ca7605e0485f4493204b8c87).